mcclient/response.py

```diff
@@ -1,4 +1,4 @@
-import datetime
+import datetime, re
```

Comment:
```suggestion
import datetime
import re
```
That's the convention i try to follow

Reply:
Got it